README.md

```diff
@@ -255,6 +274,32 @@ Add the required kodyfire metadata to your package.json
 
 Intialize your project as a git repository and push to a public Github repo
 
+To do so, kindlly follow these steps:-
+1)Intitialsie a new repository in your GitHub and make it as public repository.
```

Comment:
Let's use a numbered markdown list here instead of manually typing the bullet numbers.